telegram-bot/main.go

```diff
@@ -47,7 +45,7 @@ func tweet() {
 
 	demux := twitter.NewSwitchDemux()
 	demux.Tweet = func(tweet *twitter.Tweet) {
-		fmt.Println(tweet.IDStr)
+		log.Println(tweet.IDStr)
```

Comment:
`log` prints out to `stderr` but this is not needed in `stderr` so the use of fmt is fine.

```suggestion
		fmt.Println(tweet.IDStr)
```